docs/configuration.md

```diff
@@ -14,44 +14,118 @@ Telliot commands and config file options are as the following:
 
 #### Telliot Commands
 
-* `--logConfig` \(location of logging config file; default path is current directory\)
-* `mine` \(indicates to run the miner\)
-* `mine -r` \(indicates to mine utilizing a remote server\)
-* `dataserver` \(indicates to run the dataServer \(no mining\)\)
-* `transfer` \(AMOUNT\) \(TOADDRESS\) \(indicates transfer, toAddress is Ethereum address and amount is number of Tributes \(eg. transfer 10 0xea... \(this transfers 10 tokens\)\)\)
-* `approve` \(AMOUNT\) \(TOADDRESS\) \(amount to approve the toaddress to send this amount of tokens
-* `stake deposit` \(indicates to deposit tokens in the contract\)
-* `stake request` \(indicates you wish to withdraw your stake\)
-* `stake withdraw` \(withdraws your stake, run 1 week after request\)
-* `stake status` \(shows your staking balance\)
-* `balance` \(shows your balance\)
+
+* `accounts`   \(Show accounts\)
+
+
+* `approve` \(Address\)\(Amount\)\(Account\(optional\)\)  \(Approve tokens\)
```

Comment:
Hm, I don't think I like this format. Is there no standard formating for cli flags/options we can use here? Not sure how Thanos or the other open source project display the cli args.

Reply:
Looks like Thanos uses [embedmd](https://github.com/campoy/embedmd) to embed cli outputs in the docs wherever needed, like this one: https://github.com/thanos-io/thanos/blob/main/docs/components/store.md#flags

Reply:
yes that looks ok, lets use that format.